lib/index.js

```diff
@@ -145,6 +145,33 @@ function convertGlobPaths(globs) {
 }
 
 /**
+ * Populates any missing required properties on a Swagger object
+ * @function
+ * @param {object} swaggerObject - Swagger Object
+ * @returns {object} Swagger Object with required properties set
+ */
+function populateMissingProperties(swaggerObject) {
+  if (!swaggerObject.hasOwnProperty('paths')) {
```

Comment:
Just wondering, since these checks for creating empty objects in properties are repetitive. Wouldn't it be ok to make an array of required properties that are attached iteratively if not available?